invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/DocumentCover.js

```diff
@@ -9,19 +9,26 @@ export class DocumentCover extends Component {
     return (
       <ILSImagePlaceholder
         isLoading={this.props.isLoading}
-        style={{ width: 350, height: 500 }}
+        style={
+          this.props.placeholderStyle
+            ? this.props.placeholderStyle
+            : { width: 350, height: 500 }
```

Comment:
Do these numbers work for most layouts?

Reply:
Yes, the Image size is also fixed in semantic. I didn't notice anything wrong with this for now, but maybe it makes sense to put those in some config, to be easily changeable